src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/health/HealthCheck.kt

```diff
@@ -12,7 +12,7 @@ abstract class HealthCheck(private val webClient: WebClient, private val timeout
       .uri("/health/ping")
       .retrieve()
       .toEntity(String::class.java)
-      .block(timeout)
+      .block(timeout) ?: throw Exception("/health/ping did not return a response")
```

Comment:
Wonder if we can cover this case in `HealthCheckTest`?

Reply:
Good point. I'm not sure sure either how to test it 😆 Good to include but don't think there's much benefit of labouring over adding a test so will approve & merge